src/lib/join_record.ts

```diff
@@ -35,14 +35,17 @@ class JoinRecordS3 {
   // submission: A Join Form Submission. We append a few things to this.
   // key: The S3 path we store the submission at
   // responseCode: If we have a response code for this submission, add it here.
-  async save(joinRecord: JoinRecord, key: string = ""): Promise<string> {
+  async save(
+    joinRecord: JoinRecord,
+    preSubmission: boolean = false,
+  ): Promise<string> {
     // Get the date to store this submission under (this is part of the path)
     const submissionKey = joinRecord.submission_time
       .replace(/[-:T]/g, "/")
       .slice(0, 19);
```

Comment:
Are we planning to add the UUID here to avoid collisions? If so could we consider only adding a few characters worth of it to prevent it getting super long

Reply:
I was not planning to. Might not be a terrible idea though.